src/resources/js/commerce/gateway/stripe/checkout.js

```diff
@@ -206,60 +202,84 @@ tribe.tickets.commerce.gateway.stripe.checkout = {};
 	};
 
 	/**
-	 * Submit the payment to stripe code.
+	 * Submit the payment to Stripe for Payment Element.
+	 *
+	 * @since TBD
 	 *
 	 * @param {String} order The order object returned from the server.
 	 *
 	 * @return {Promise<*>}
 	 */
 	obj.submitMultiPayment = async ( order ) => {
-		var elements = obj.stripeElements;
-		var billing_details = billing.getDetails(false);
-		var order = order;
+		const billingDetails = billing.getDetails( false );
+
 		return obj.stripeLib.confirmPayment( {
-			elements,
+			elements: obj.stripeElements,
 			redirect: 'if_required',
 			confirmParams: {
 				return_url: order.redirect_url,
 				payment_method_data: {
-					billing_details: billing_details
+					billing_details: billingDetails
 				}
 			}
-		} ).then( function( result ) {
-			obj.submitButton(true);
-			if ( result.error ) {
-				obj.handlePaymentError( result );
-			} else {
-				if ( result.paymentIntent.status === 'succeeded' ) {
-					obj.handlePaymentSuccess( result );
-				}
+		} ).then( obj.handleConfirmPayment );
+	};
+
+	/**
+	 * Handle the confirmation of the Payment on PaymentElement.
+	 *
+	 * @since TBD
+	 *
+	 * @param result
+	 */
+	obj.handleConfirmPayment = ( result ) => {
+		obj.submitButton( true );
+		if ( result.error ) {
+			obj.handlePaymentError( result );
+		} else {
+			// @todo feels like we are missing a error handling here.
```

Comment:
no exactly error handling, but some cases in which the payment method returns a partial success, like a status of `pending_confirm` and some others.